src/OnboardingSPA/pages/Steps/DesignThemeStyles/Menu/index.js

```diff
@@ -116,10 +146,17 @@ const StepDesignThemeStylesMenu = () => {
 		<DesignStateHandler>
 			<CommonLayout>
 				<div className="theme-styles-menu">
-					<HeadingWithSubHeading
-						title={ currentStep?.heading }
-						subtitle={ currentStep?.subheading }
-					/>
+					<HeadingWithSubHeading title={ currentStep?.heading }>
+						<h3 className="nfd-main-heading__subtitle">
+							{ currentStep?.subheading }{ ' ' }
+							<button
+								className="theme-styles-menu__custom-pages-link"
+								onClick={ skiptoCustomPage }
+							>
+								build a custom design.
```

Comment:
Should we pass this to the `__` function? Probably a content.js file, Ref: https://github.com/newfold-labs/wp-module-onboarding/pull/226/files#diff-b7a84cc60ee7823fada85d261435b4bd50e415622216546b4ffbdfed7ac9557c

Reply:
Won't a **js** file for just a **Heading** and **Subheading** be an overkill?

Reply:
Done.